prov/efa/src/rxr/rxr.h

```diff
@@ -189,7 +189,6 @@ extern struct util_prov rxr_util_prov;
 extern struct efa_ep_addr *local_efa_addr;
 
 struct rxr_env {
-	int shm_cma_cap;
```

Comment:
shm_cma_cap should not have been introduced at all.

Reply:
We introduced this env. variable to check if we need mr registration in the last commit.

Reply:
Is it used in last commit? If not, maybe we remove it from last commit?

Reply:
It is in the last commit. 
```/c
} else if (rxr_env.enable_shm_transfer && peer->is_local && rxr_env.shm_cma_cap) {
        for (i = 0; i < tx_entry->iov_count; ++i)
                read_iov[i].key = 0;
```